experimental/segmenter/src/line_breaker.rs

```diff
@@ -503,7 +524,11 @@ impl<'l, 's, Y: LineBreakType<'l, 's>> Iterator for LineBreakIterator<'l, 's, Y>
             }
 
             // If break_state is equals or grater than 0, it is alias of property.
-            let mut break_state = get_break_state(left_prop, right_prop);
+            let mut break_state = get_break_state_from_table(
+                &self.segmenter.payload.get().rule_table,
```

Comment:
> However, the bench "Line Break/Latin1/En" is about 30% slower, which is unexpected. I don't know whether I made some coding error or whether the compiler was doing some sort of optimization before that it can't do now.

Getting `rule_table` and `property_table` repeatedly in `next()` might be the root cause of the performance downgrade. I quickly hack a patch storing tables in `LineBreakIterator` directly https://gist.github.com/aethanyc/59858b25ce1a0b9c1f75f1053844de83, and it does restore the performance to it was. Could you try caching the table from the payload and see if you see similar performance improvements? (I tried, but somehow struggling with the lifetime parameters ...) Can be done here or in a follow up PR.